app/Models/QueryBuilders/SearchFormQueryBuilder.php

```diff
@@ -111,16 +145,35 @@ public function buildQuery(): array
         $orQueries = [];
         $andQueries = [];
         $boolQueries = [];
+        $dateRange = [];
+
+        if (isset($this->startDate) && !empty($this->startDate)) {
+            $carbonStartDate = new Carbon($this->startDate);
+            $dateRange['gte'] = $carbonStartDate->toAtomString();
+        }
+
+        if (isset($this->endDate) && !empty($this->endDate)) {
```

Comment:
Do we need both isset and empty here?

Reply:
Yes, as I already mentioned above,
"isset" checks if user want to search for date
"empty" checks if date is not empty string ''.

Reply:
Same here.

Reply:
Removed check for "isset".